graphql/schema.graphql

```diff
@@ -498,6 +498,7 @@ type Volume {
 
 type PatchProject {
   variants: [ProjectBuildVariant!]!
+  # note the tasks field is deprecated and can be removed whenever
   tasks: [String!]!
```

Comment:
We can remove  this from the schema in this pr rn if we deploy the front end portion of this first.

Reply:
it will still break for those that loaded the patch page before the deploy, so it's better to do it separately

Reply:
As long as we deploy the front end before this pr is deployed with that field omitted we can safely remove this field. As long as a user refreshes their page between the time we deploy the front end and the time we deploy the back end changes they wouldn't notice any issues. Even if they do have an older version of the site open any errors would be resolved with a refresh.

Reply:
Yep that's the situation I'm trying to avoid since it'll be annoying for those who do see the error. As a small added bonus we don't need to coordinate deploys if we do it this way